bin/create-pull-secret

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [[ -z "${VAULT_TOKEN}" ]]; then
```

Comment:
can we document what this value should be?

Reply:
fixed